lib/modules/forgot_password/forgot_password_presenter.dart

```diff
@@ -0,0 +1,16 @@
+part of 'forgot_password_module.dart';
+
+abstract class ForgotPasswordPresenter extends Presenter<ForgotPasswordView,
+    ForgotPasswordInteractor, ForgotPasswordRouter> {}
+
+class ForgotPasswordPresenterImpl extends ForgotPasswordPresenter
+    implements ForgotPasswordViewDelegate, ForgotPasswordInteractorDelegate {
+  @override
+  final emailTextFieldDidChange = BehaviorSubject<String>();
+
+  @override
+  final resetButtonDidTap = BehaviorSubject<String>();
+
+  @override
+  final alertDialogDidClose = BehaviorSubject<void>();
+}
```

Comment:
@markgravity did you forget adding `.addTo(disposeBag)` for these vars? 👀

Reply:
@manh-t Thanks! I updated it in [here](https://github.com/markgravity/flutter-survey/blob/5b617d3b2fcbc0ede0a43641b6ed78c14f2d2c4a/lib/modules/forgot_password/forgot_password_presenter.dart#L9)